sigmastate/src/main/scala/sigmastate/utxo/transformers.scala

```diff
@@ -9,12 +9,27 @@ import sigmastate.serialization.OpCodes
 import org.ergoplatform.ErgoBox.RegisterId
 import sigmastate.Operations._
 import sigmastate.lang.exceptions.InterpreterException
+import special.sigma.Box
 
 
+// TODO refactor: remove this trait as it doesn't have semantic meaning
```

Comment:
What does have then? Or what does the TODO mean if no replacement?

Reply:
removed this TODO